src/main/scala/cromiam/server/CromIamServer.scala

```diff
@@ -1,33 +1,38 @@
-package server
+package cromiam.server
 
 import akka.Done
 import akka.actor.ActorSystem
 import akka.event.Logging
 import akka.http.scaladsl.server.{HttpApp, Route}
 import akka.http.scaladsl.settings.ServerSettings
 import akka.stream.ActorMaterializer
-import com.typesafe.config.{Config, ConfigFactory}
-import webservice.{CromIamApiService, SwaggerService}
+import cats.data.Validated.{Invalid, Valid}
+import com.typesafe.config.ConfigFactory
+import cromiam.server.config.CromIamServerConfig
+import cromiam.webservice.{CromIamApiService, SwaggerService}
 
 import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future, Promise}
 
 
 object CromIamServer extends HttpApp with CromIamApiService with SwaggerService {
 
+  final val configuration: CromIamServerConfig = CromIamServerConfig.getFromConfig(ConfigFactory.load()) match {
+    case Valid(c) => c
+    case Invalid(errors) => throw new Exception("Bad CromIAM configuration:" + errors.toList.mkString("\n", "\n", "\n"))
+  }
+
   def run(): Unit = {
-    CromIamServer.startServer(CromIamServer.config.getString("http.interface"), CromIamServer.config.getInt("http.port"), ServerSettings(CromIamServer.config))
+    CromIamServer.startServer(configuration.cromIamConfig.http.interface, configuration.cromIamConfig.http.port, ServerSettings(""))
```

Comment:
Hmm... Not for the set of options currently in `application.conf`, but I guess we should support people adding other values to the `http` block if they want to. I'll try to re-wire them through